src/components/About/styles.scss

```diff
@@ -0,0 +1,148 @@
+// Mobile portrait
+.wrapper {
+	display: flex;
+	width: 100%;
+	height: 100%;
+	flex-direction: column;
+	background-color: blue;
+}
+
+	.hero {
+		display: block;
+		width: 100%;
+		height: 38%;
+	}
+
+	.textContainer {
+		width: 100%;
+		height: 62%;
+		--padding__body: 10vw;
```

Comment:
let's put all the variables as the very last thing in a class declaration - they look so different and weird that it feels odd to have them mixed in.